CITATION.cff

```diff
@@ -239,6 +243,10 @@ preferred-citation:
   start: 1896
   doi: 10.21105/joss.01896
   authors:
+    - given-names: Berk
+      family-names: Gerçek
+      affiliation: 'University of Geneva, Department of Fundamental Neuroscience'
+      orcid: 'https://orcid.org/0000-0003-1063-6769'
```

Comment:
I don't think you should add this here; you can't retroactively become an author on a published paper :)

Reply:
No I think it's fine, this is for citing **the software**, no?

Reply:
I think it only applies here:

<img width="546" alt="Screenshot 2025-01-17 at 09 24 59" src="https://github.com/user-attachments/assets/2264ad5f-0835-41b9-ae40-5ae8a62fba33" />

Reply:
In the citation CFF file there are two sections of authors. One for the paper (do not add yourself there), and one for the software (please add yourself there).

Reply:
@hoechenberger did you look at the diff? This comment is in the section for the 2019 joss paper. There's a separate, correct addition of @berkgercek's info above, in the section for the software itself.

Reply:
Thanks for the clarification, I did not realize this!

Reply:
Thanks! I'll remove myself from the published paper's section, apologies for the mistake